clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -386,7 +387,8 @@ public enum Errors {
     STALE_MEMBER_EPOCH(113, "The member epoch is stale. The member must retry after receiving its updated member epoch via the ConsumerGroupHeartbeat API.", StaleMemberEpochException::new),
     MISMATCHED_ENDPOINT_TYPE(114, "The request was sent to an endpoint of the wrong type.", MismatchedEndpointTypeException::new),
     UNSUPPORTED_ENDPOINT_TYPE(115, "This endpoint type is not supported yet.", UnsupportedEndpointTypeException::new),
-    UNKNOWN_CONTROLLER_ID(116, "This controller ID is not known.", UnknownControllerIdException::new);
+    UNKNOWN_CONTROLLER_ID(116, "This controller ID is not known.", UnknownControllerIdException::new),
+    REQUEST_LIMIT_REACHED(117, "The request has reached the limit.", RequestLimitReachedException::new);
```

Comment:
The phrasing is very vague. I guess it is a request _size_ limit?

Reply:
Yes. Updated the comment.